app/config/config.yml

```diff
@@ -44,12 +44,12 @@ assetic:
     debug:          "%kernel.debug%"
     use_controller: false
     bundles:        [ ]
-    #java: /usr/bin/java
     filters:
         cssrewrite: ~
-        sass:
-            bin:      "/usr/bin/sass"
-            apply_to: "\.scss$"
+        jsqueeze: ~
+        scssphp:
+            # the formatter must be the FQCN (don't use the 'compressed' value)
```

Comment:
indeed. the ScssPhp library expects a class name. It is always nice to see consistency between libraries created by the same dev :smile: